gizmo.py

```diff
@@ -0,0 +1,55 @@
+class Gizmo:
+    def __init__(self, name):
+        self.name = name
+
+    def speak(self):
+        print (self.name)
+
+def hello(name, country = "finland"):
+    print("Hello ",name,", how are things in ",country,"?")
```

Comment:
By default, the `print` function will put spaces between the different parts that you give it, hence this will print:
```python
"Hello  {name} , how are things in  {country} ?"
```

See if you can get rid of these extra spaces.